e/error-prone-annotations/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "pratibh-ibm",
+    "package_name": "error-prone-annotations",
```

Comment:
Package name should match the repo name. Accordingly, please update the folder name, script name and anywhere else applicable.